server/schedule/operator/create_operator.go

```diff
@@ -177,3 +178,52 @@ func CreateScatterRegionOperator(desc string, cluster opt.Cluster, origin *core.
 		EnableLightWeight().
 		Build(0)
 }
+
+// CreateLeaveJointStateOperator creates an operator that let region leave joint state.
+func CreateLeaveJointStateOperator(desc string, cluster opt.Cluster, origin *core.RegionInfo) (*Operator, error) {
+	b := newBuilderWithBasicCheck(desc, cluster, origin)
+
+	if b.err == nil && !core.IsInJointState(origin.GetPeers()...) {
+		b.err = errors.Errorf("cannot build leave joint state operator for region which is not in joint state")
+	}
```

Comment:
why not use NewBuilder

Reply:
for skip joint state check

Reply:
A better way can be adjusted in the next PR.

Reply:
oh, got it, made a mistake.